src/library_glfw.js

```diff
@@ -467,19 +467,23 @@ var LibraryGLFW = {
 
       if (event.target != Module["canvas"] || !GLFW.active.cursorEnterFunc) return;
 
+      if (GLFW.active.cursorEnterFunc) {
 #if USE_GLFW == 3
-      {{{ makeDynCall('vii', 'GLFW.active.cursorEnterFunc') }}}(GLFW.active.id, 1);
+        {{{ makeDynCall('vii', 'GLFW.active.cursorEnterFunc') }}}(GLFW.active.id, 1);
 #endif
+      }
     },
 
     onMouseleave: function(event) {
       if (!GLFW.active) return;
 
       if (event.target != Module["canvas"] || !GLFW.active.cursorEnterFunc) return;
```

Comment:
I guess you can drop the second part of this condition now?

Reply:
Good catch, done.